cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal

```
@@ -337,7 +368,10 @@ function createDirectories(string directoryPath) returns (boolean) {
 function closeChannel(io:ByteChannel byteChannel) {
     match byteChannel.close() {
         error channelCloseError => {
-            io:println(logger.formatLog("Error occured while closing the channel: " + channelCloseError.message));
+            error channelError = {
+                message: logFormatter.formatLog("Error occured while closing the channel: " + channelCloseError.message)
+            };
+            throw channelError;
```

Should we throw an error here? Will a `channelCloseError` break the flow?